homeassistant/components/tibber/sensor.py

```diff
@@ -428,6 +428,7 @@ async def _fetch_data(self) -> None:
             await self._tibber_home.update_info_and_price_info()
         except (TimeoutError, aiohttp.ClientError):
             return
+        self._spread_load_constant = randrange(TWENTY_MINUTES)  # new spread next day
```

Comment:
yes, I know but thought it could be a good idea that the spread is not set in stone forever since starting the component.
So every user would have another chance getting the prices earlier the next day.

There is also another aspect which I saw today in the log with that new approach running on my system (together with the new price attributes of my previous PR):

The first fetch for new data was at 13:02 with spread 1 min (set yesterday).
New spread set to 16 min.
Unfortunately new data at 13:02 was not ready, means Tibber delivered still old "today" data.

Then the next fetch was at 13:17 (with the spread 16 min)
This fetch worked fine so with "tomorrow" data.
Without the new spread it would directly follow to this every minute fetch mode until "tomorrow" is available which cloud be more payload ...

Reply:
> So every user would have another chance getting the prices earlier the next day.

No, it is not more likely to get price first if your spread constant is 1min or 10min.  (Maybe a bit less likely to get price first if the constant is 20 min)
Since when price are available for each customer is pretty random. Both when Nordpool publishes the prices and when each user cache is updated at Tibber.
I would suggest you remove this change.

Reply:
I only thought that it could be a benefit when the spread is not set in stone.